addons/mrp/views/mrp_config_settings_views.xml

```diff
@@ -64,29 +64,13 @@
                                 <div class="o_setting_right_pane">
                                     <label for="module_mrp_maintenance"/>
                                     <div class="text-muted">
-                                        Schedule maintenance activities
+                                        Schedule the maintenance of your equipment
```

Comment:
LAP forgot to put an s at the end of equipment. It should be "Schedule the maintenance of your equipments".